app/src/main/res/values/strings.xml

```diff
@@ -304,55 +304,55 @@
     <string name="shared_texts">Shared Texts</string>
     <string name="github" translatable="false">GitHub</string>
     <string name="join_us">Join Us</string>
-    <string name="visit_license_url">Visit License URL</string>
-    <string name="visit_web_page">Visit Web Page</string>
+    <string name="visit_license_url">Go to License URL</string>
+    <string name="visit_web_page">Visit Website</string>
     <string name="background_service">Background Service</string>
     <string name="empty_clients_list">Clients appear here</string>
     <string name="all_clients">All Clients</string>
     <string name="connect_to_wifi">Connect to Wi-Fi</string>
-    <string name="wifi_connection_notice">1- Open Wi-Fi settings
-        \n\n2- Wait until \'%s\' appears on the Wi-Fi list
-        \n\n3- Pick it (%s) and paste the password (%s)
-        \n\n4- Return here after successful connection
+    <string name="wifi_connection_notice">1- Open Wi-Fi settings.
+        \n\n2- Wait until \'%s\' appears on the Wi-Fi list.
+        \n\n3- Pick it (%s) and paste the password (%s).
+        \n\n4- Return here after connecting.
     </string>
-    <string name="copy_password_and_open_settings"><![CDATA[Copy Password & Open Settings]]></string>
+    <string name="copy_password_and_open_settings"><![CDATA[Copy Password adn Open Settings]]></string>
```

Comment:
`Copy Password &amp; Open Settings` will be the correct use here.

Reply:
```suggestion
    <string name="copy_password_and_open_settings"><![CDATA[Copy Password and Open Settings]]></string>
```

Reply:
I don't understand how that isn't a wrong use of an ampersand.

Reply:
Which one do you mean? Ampersand is one of the special characters for some reason, so it should either be used as in the original or the one I suggested.

Reply:
I mean it is not interchangable with "and" linguistically, even though there is some overlap.